security/src/test/groovy/io/micronaut/docs/security/authorization/X509AuthorizationSpec.groovy

```diff
@@ -42,16 +43,21 @@ class X509AuthorizationSpec extends Specification {
             ts.store(os, "123456".toCharArray())
         }
 
+        def filePathPrefix = 'file://'
+        if (OperatingSystem.current.family == OperatingSystem.Family.WINDOWS) {
+            filePathPrefix = 'file:'
+        }
+
         EmbeddedServer server = ApplicationContext.run(EmbeddedServer, [
                 'spec.name': X509AuthorizationSpec.simpleName,
                 'micronaut.ssl.enabled': true,
                 'micronaut.server.ssl.port': -1,
-                'micronaut.ssl.build-self-signed': false,
+                'micronaut.server.ssl.build-self-signed': false,
```

Comment:
`micronaut.ssl.build-self-signed` is deprecated. need to use `micronaut.server.ssl.build-self-signed`